src/tunacode/ui/commands/resume.py

```
@@ -0,0 +1,143 @@
+
+        sessions = app.state_manager.list_sessions()
+        if not sessions:
+            app.notify("No saved sessions found")
+            return
+
+        current_session_id = app.state_manager.session.session_id
+
+        def on_session_selected(session_id: str | None) -> None:
+            if not session_id:
+                return
+            asyncio.create_task(self._load_session(app, session_id, sessions))
```

Ruff (0.15.0)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)